# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipsvd/patches/twoftpd.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./twoftpd-anon-conf.c.orig	2004-05-10 11:00:38.000000000 -0400
+++ ./twoftpd-anon-conf.c	2004-05-10 11:06:23.000000000 -0400
@@ -92,15 +92,15 @@
 	    "exec 2>&1\n"
 	    "umask 022\n"
 	    "exec \\\n"
-	    "tcpserver -DRHv -llocalhost ", ip, " 21 \\\n"
+	    "tcpsvd -vv -llocalhost ", ip, " 21 \\\n"
 	    "softlimit -m 2000000 \\\n"
 	    "envdir ", maindir, "/env \\\n",
 	    conf_bin, "/twoftpd-anon");
   make_file("log/run", 0755,
 	    "#!/bin/sh\n"
 	    "exec \\\n"
 	    "setuidgid ", logname, " \\\n"
-	    "multilog t ", logdir, 0, 0, 0);
+	    "svlogd -t ", logdir, 0, 0, 0);
   make_fileu("env/CHROOT", 1);
   make_fileu("env/GID", ftpgid);
   make_file("env/HOME", 0644, ftpdir, 0, 0, 0, 0, 0, 0);
--- ./twoftpd-conf.c.orig	2004-05-10 11:00:38.000000000 -0400
+++ ./twoftpd-conf.c	2004-05-10 11:05:28.000000000 -0400
@@ -84,7 +84,7 @@
 	     "exec 2>&1\n"
 	     "umask 022\n"
 	     "exec \\\n"
-	     "tcpserver -DRHv -llocalhost ", ip, " 21 \\\n"
+	     "tcpsvd -vv -llocalhost ", ip, " 21 \\\n"
 	     "envdir ", maindir, "/env \\\n");
   obuf_put7s(&conf_out,
 	     "softlimit -m 2000000 \\\n",
@@ -96,8 +96,8 @@
   make_file("log/run", 0755,
 	    "#!/bin/sh\n"
 	    "exec \\\n"
	    "setuidgid ", logacct->pw_name, " \\\n"
-	    "multilog t ", logdir, 0, 0, 0);
+	    "svlogd -t ", logdir, 0, 0, 0);
   
   if (dochroot) make_fileu("env/CHROOT", 1);
   
